helix-term/src/keymap/default.rs

```diff
@@ -205,6 +205,7 @@ pub fn default() -> HashMap<Mode, Keymap> {
         "space" => { "Space"
             "f" => file_picker,
             "F" => file_picker_in_current_directory,
+            "v" => file_picker_in_current_view_directory,
```

Comment:
I would prefer to start with no binding for now. The space keymap is already very crowded and it seems like the file submap could use some more discussion

Reply:
sounds reasonable